mautic/templates/deployment.yaml

```diff
@@ -11,6 +15,10 @@ spec:
       {{- include "mautic.selectorLabels" . | nindent 6 }}
   template:
     metadata:
+      {{- with .Values.deployment.annotations }}
```

Comment:
The values here is related to deployment template, lets rename the value to `deploymentTemplate`.

For case with osiris, `deployment.annotations` and `deploymentTemplate.annotations` will be same.